src/test/scala/sigmastate/SoftForkabilitySpecification.scala

```diff
@@ -313,4 +312,29 @@ class SoftForkabilitySpecification extends SigmaTestingData {
     })
   }
 
+  property("CheckCostFuncOperation rule") {
+    val exp = Height
+    val v2vs = vs.updated(CheckCostFuncOperation.id,
+      ChangedRule(CheckCostFuncOperation.encodeVLQUShort(Seq(Height.opCode))))
+    checkRule(CheckCostFuncOperation, v2vs, {
+      val IR.Pair(calcF, _) = IR.doCostingEx(emptyEnv, exp, okRemoveIsProven = false)
+      // use calcF as costing function to have forbidden (not allowed) op (Height) in the costing function
+      CheckCostFunc(IR)(IR.asRep[Any => Int](calcF)) { }
+    })
+  }
+
+  property("CheckCostFuncOperation rule (OpCodeExtra") {
```

Comment:
how this test is different from the previous?
Looks like it is checking the same, so it doesn't improve code coverage.

Reply:
It tests that "OpCodeExtra" op can be enabled. First one tested "OpCode" op.